orangecontrib/geo/widgets/owgeotransform.py

```diff
@@ -102,18 +114,15 @@ def __init__(self):
                                         self.apply)
 
     def init_attr_values(self):
-        self.variable_model.set_domain(self.data.domain if self.data else None)
         self.Error.no_lat_lon_vars.clear()
+        self.variable_model.set_domain(self.data.domain if self.data else None)
 
-        if self.variable_model.rowCount() < 2:
-            self.Error.no_lat_lon_vars()
-            lat, lon = None, None
-            self.data = None
-        else:
-            lat, lon = find_lat_lon(
-                self.data, filter_hidden=True, fallback=False)
-            if lat is None or lon is None:
-                lat, lon = self.variable_model[:2]
+        lat, lon = None, None
+        if self.data:
+            lat, lon = find_lat_lon(self.data, filter_hidden=True)
+            if not (lat and lon):
+                self.Error.no_lat_lon_vars()
+                self.data = None
```

Comment:
This should probably update self.variable_model. If `lat, lon = None, None`, the model